src/react/workflow/ExpirationForm.tsx

```diff
@@ -134,197 +182,185 @@ export function ExpirationForm({ bucketList, locations, prefix = '' }: Props) {
-              </T.Row>
-              <T.Row
-                style={{
-                  maxHeight: 'initial',
-                  height: '100%',
-                  marginTop: '2rem',
-                  alignItems: 'baseline',
+                  );
                 }}
-              >
-                <T.Key>Tags</T.Key>
-                <T.Value>
-                  <Controller
-                    name={`${prefix}filter.objectTags`}
-                    control={control}
-                    defaultValue={[{ key: '', value: '' }]}
-                    render={({ field: { onChange, value } }) => {
-                      return (
-                        <TagsFilter
-                          handleChange={onChange}
-                          control={control}
-                          fieldName={`${prefix}filter.objectTags`}
-                          tags={value}
-                          watch={watch}
-                        />
-                      );
-                    }}
-                  />
-                </T.Value>
-              </T.Row>
-              <T.Row>
-                <T.ErrorContainer>
-                  <ErrorInput
-                    id="error-prefix"
-                    error={errors[`${prefix}filter.objectTags`]?.message}
-                  />
-                </T.ErrorContainer>
-              </T.Row>
-            </T.GroupContent>
-          </T.Group>
+              />
+            }
+          />
+        </FormSection>
 
-          <T.Group>
-            <T.GroupName>Action</T.GroupName>
-            <T.GroupContent>
-              <T.Row data-testid="toggle-action-expire-current-version">
-                <T.KeyTooltip
-                  tooltipMessage={`
+        <FormSection
+          title={{ name: 'Action' }}
+          forceLabelWidth={forceLabelWidth}
+        >
+          <T.GroupContent required>
+            <T.Row data-testid="toggle-action-expire-current-version">
+              <T.KeyTooltip
+                tooltipMessage={`
                     If the bucket is versioned, a delete marker will be added on objects older than ${
                       currentVersionTriggerDelayDays || 'provided'
                     } days. This does not free up storage space.
                     If the bucket is not versioned, the objects older than ${
                       currentVersionTriggerDelayDays || 'provided'
                     } days will be permanently removed.
                     `}
-                  tooltipWidth="13rem"
-                  size={60}
-                >
-                  {' '}
-                  Expire <span style={{ fontWeight: 'bold' }}>
-                    Current
-                  </span>{' '}
-                  version of objects{' '}
-                </T.KeyTooltip>
-                <T.Value
+                tooltipWidth="13rem"
+                size={60}
+              >
+                Expire <span style={{ fontWeight: 'bold' }}>Current</span>{' '}
```

Comment:
We introduced the lately the Text component in core-ui.
```suggestion
                Expire <Text isEmphazed>Current</Text>{' '}
```